tests/run_tests

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
```

Comment:
To make this file executable, run `chmod +x tests/run_tests`. Then, all you need run the test in the command line is `tests/run_tests`.